kolibri/content/serializers.py

```diff
@@ -40,12 +40,18 @@ def to_representation(self, instance):
             on_device_resources = descendants.exclude(kind=content_kinds.TOPIC).filter(available=True).count()
             on_device_file_size = local_files.filter(available=True).aggregate(Sum('file_size'))['file_size__sum'] or 0
 
+            num_coach_contents = ContentNodeGranularSerializer(
```

Comment:
I'm not sure if I'm getting it right, but I feel like calling the ContentNodeGranularSerializer to only get `num_coach_contents` results in calling other functions in ContentNodeGranularSerializer which we don't need? I'm thinking that it could run faster if we get a copy of the function `get_num_coach_contents` in the ChannelMetadataSerialize and call it directly from there?

Reply:
looks good! @jonboiser thanks for the update